connector/dao/src/main/java/it/pagopa/selfcare/mscore/connector/dao/UserConnectorImpl.java

```diff
@@ -70,6 +72,19 @@ public List<OnboardedUser> findAll() {
         return repository.findAll().stream().map(userMapper::toOnboardedUser).collect(Collectors.toList());
     }
 
+    @Override
+    public List<OnboardedUser> findAll(Integer page, Integer size, String productId) {
```

Comment:
you could add some reference about status that you query (ACTIVE,DELETED and SUSPENDED)....for example findAllValidUser and add some comments in java doc description where you explain that the query find only ACTIVE,DELETED and SUSPENDED status

Reply:
Would it be better to pass the set of statuses that the query needs to filter for as a parameter to the method? or Can i define it as a constant in the class and pass it straight into the query?

Reply:
you can change the method naming findAllByStatus adding the list of status as parameter or just change the method name findAllValid but adding as comment that the query find only ACTIVE,DELETED and SUSPENDED status